lib/grape-swagger/endpoint.rb

```diff
@@ -4,6 +4,7 @@
 require 'active_support/core_ext/string/inflections.rb'
 
 module Grape
+  # rubocop:disable Metrics/ClassLength
```

Comment:
please don't add it here, specify it in the .rubocop-todo.yml

Reply:
okay, and I will fix the other two rubocop issues that causes the CI to fail.